app.py

```diff
@@ -103,8 +110,12 @@ def get_user_names(array_of_user_ids, logger, client):
     return names
 
 def refresh_home_tab(client, user_id, logger, top_message, team_id, context):
-    print(client)
-    print(context)
+    print("CLIENT", client)
```

Comment:
Honestly, we can take a lot of these print statements out at some point... I put them in during early debugging, and was too lazy to take them back out. Some of them like client and context don't even return anything interesting